### PR TITLE
Add recipe for mobie python package

### PR DESCRIPTION
**Pull request by tischi**
Dear conda-forge team,

I would like to add the mobie python package, which is a wrapper around the [MoBIE Java image viewer](https://mobie.github.io/).

Thank you for reviewing this PR!

Best, Tischi 

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/mobie/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  imports:
+    - mobie
+  commands:
+    - mobie --help
+
+about:
+  home: "https://github.com/mobie/mobie-viewer-fiji"
+  license: BSD
```
Comment (dopplershift) [inline recipes/mobie/meta.yaml:38]:
```suggestion
  license: BSD-2-clause
```

File: recipes/mobie/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - maven
+
+test:
+  imports:
+    - mobie
+  commands:
+    - mobie --help
+
+about:
+  home: "https://github.com/mobie/mobie-viewer-fiji"
```
Comment (dopplershift) [inline recipes/mobie/meta.yaml:37]: The package metadata was wrong, this is the correct URL:
```suggestion
  home: "https://github.com/mobie/mobie-cmd"
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple minor tweaks and this is good to go.

**Comment by tischi** [comment]
@dopplershift 
Thank you so much ❤️ ! 
I committed your suggested changes!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**